src/app.js

```diff
@@ -214,8 +214,13 @@ function run () {
     },
     getValue: (cb) => {
       try {
-        var comp = $('#value').val().split(' ')
-        cb(null, executionContext.web3().toWei(comp[0], comp.slice(1).join(' ')))
+        var number = document.querySelector('#value').value
+        var select = document.getElementById('unit')
+        var index = select.selectedIndex
+        var selectedUnit = select.querySelectorAll('option')[index].innerHTML
+        var unit = ''
+        if (selectedUnit === 'wei') { unit = 'wei' }
```

Comment:
Sorry to be so picky here, but since this is so delicate: can you please change the unit selection to explicitly ask for ``ether``, and switch to ``wei`` for all other occasions?

```
if (selectedUnit === 'ether') {
  unit = 'ether'
} else {
  unit = 'wei'
}
```

Otherwise if something goes wrong with the ``#value``  selection (let's say e.g. someone accidentally introduces another ``#value`` field in the future or JS for whatever reasons can't read it from the ``select`` field) this falls again back to ``ether``.

Reply:
I would also suggest to read the choice from the ``id`` field with ``querySelectorAll('option')[0].id`` instead of using ``innerHTML``.

Then this is more stable regards to renaming, if you e.g. decide to change ``wei`` to ``Wei``.

Reply:
And one last thing (then I'll stop, I promise :-): just thought about it a bit, it would be probably nice (since this is closer to real world number inputs) to also use one unit from the middle, here is an overview: http://ethdocs.org/en/latest/ether.html (when you see the number of zeros you generally see why this is so dangerous stuff... :-))

I think also taking ``Gwei`` would make sense, this would be ``unit = 'gwei' in addition.

Reply:
+1 for Gwei . Here is a list of units: https://etherconverter.online . I guess ether, wei, gwei is sufficient

Reply:
Thanks you both for the review. Makes sense, if-else, not using inerHTML and also this with other units. Let me do the fixes.

@yann300 For the new unit, I will make a new PR.

Reply:
![bildschirmfoto 2017-10-13 um 15 46 14](https://user-images.githubusercontent.com/931137/31549198-bdc4836a-b02d-11e7-96f7-177f90a48466.png)

When I'm minimizing the created contract boxes, I am not able to expand them again (see screenshot, MacOS, Chrome).

Reply:
Generally I think it is really important that these boxes are at least discardable/removable. This is otherwise really annoying, after 4-5 creations one has to reload the page to get rid of them.